editor/components/post-format/index.js

```diff
@@ -70,8 +71,14 @@ function PostFormat( { onUpdatePostFormat, postFormat = 'standard', suggestedFor
 export default compose( [
 	withSelect( ( select ) => {
 		const { getEditedPostAttribute, getSuggestedPostFormat } = select( 'core/editor' );
+		const format = getEditedPostAttribute( 'format' );
+		const themeSupports = select( 'core' ).getThemeSupports();
+		// Ensure current format is always in the set.
+		// The current format may not be a format supported by the theme.
+		const supportedFormats = union( [ format ], get( themeSupports, 'formats', [] ) );
 		return {
 			postFormat: getEditedPostAttribute( 'format' ),
```

Comment:
We can leverage the assigned variable `format` here.

(And optionally rename as `postFormat` for consistency / benefit of object property shorthand)

Reply:
39dbb81491c1f036fecc65ef04017dad8ab67914